### PR TITLE
add new term MONDO:0100085 cancer of long bone of upper limb

### LINKED ISSUE #899: [NTR] 'cancer of long bone of upper limb'
**Preferred term label:**
'cancer of long bone of upper limb'

**Synonyms**
malignant upper limb long bone neoplasm
malignant neoplasm of upper limb long bone
malignant neoplasm of long bones of arm
long bones of upper limb cancer

**Definition (free text, please give PubMed ID, if applicable, in format PMID:#######)**
A cancer that involves the upper limb long bone.

**Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)**
'bone caner'

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #899

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@cmungall I fixed the conflicts. This should be ready to merge (if Travis passes)

**Comment by nicolevasilevsky** [comment]
I omitted the [Term] in the edit file, and couldn't open the file in Protege, but the checks still passed.

Could we add a check for this?

image

**Comment by nicolevasilevsky** [comment]
it is fixed now

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -428289,6 +428289,23 @@ synonym: "actin myopathy" EXACT []
...
+synonym: "cancer of long bone of fore limb" EXACT []
+synonym: "cancer of long bone of forelimb" EXACT []
+synonym: "cancer of long bone of upper extremity" EXACT []
+synonym: "cancer of upper extremity long bone" EXACT []
+synonym: "long bones of upper limb cancer" EXACT []
+synonym: "malignant neoplasm of upper limb long bone" EXACT []
+synonym: "malignant upper limb long bone neoplasm" EXACT []
+is_a: MONDO:0024311 ! cancer affecting bone of limb skeleton
+intersection_of: MONDO:0004992 ! cancer
+intersection_of: disease_has_location UBERON:0002102 ! forelimb
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: you don't need this one here. 'forelimb long bone' is sufficient. And it's best to conform to the pattern in cancer.yaml which has one slop for relation

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: got it - I fixed this

**Pull request merged**